terraform/e2e/variables.tf

```diff
@@ -56,3 +56,16 @@ variable "bigquery_table_delete_protection" {
   type        = bool
   default     = false
 }
+
+variable "pmap_mapping_resource_scope" {
+  description = <<EOT
+        "The scope for where the resources resides in.
+        Options can be one of the following:
+        projects/{PROJECT_ID}
+        projects/{PROJECT_NUMBER}
+        folders/{FOLDER_NUMBER}
+        organizations/{ORGANIZATION_NUMBER}"
+    EOT
+  type        = string
+  default     = ""
```

Comment:
If this is required, why do we still default to empty?

Reply:
Because mapping and policy uses the same module. https://github.com/abcxyz/pmap/blob/main/terraform/e2e/main.tf#L30-L59

If there is no default value here, the policy module would fail. Now I default the value to `projects/project_id`

Reply:
I see. Thanks!